.github/parm/use_case_groups.json

```
@@ -122,7 +122,7 @@
   {
     "category": "s2s",
     "index_list": "1-3",
-    "run": false
+    "run": true
```

Turn off use cases for pull request so they aren't run on push for future pull requests
```suggestion
    "run": false
```